flake8_pantsbuild.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import ast
+import sys
+
+if sys.version_info >= (3, 8):
+    from importlib.metadata import version
+else:
+    from importlib_metadata import version
+
+
+PNT800 = (
```

Comment:
Note that the original code was `T800`. Flake8 strongly encourages plugin authors to use 3 letters for their codes due to reduce ambiguity and avoid possible name clashes with other plugins. 3 letter codes weren't possible until recently.

`PNT` seemed the best acronym for `Pants` that I could think of, but also considered `PAN`.

Reply:
Since this plugin has ~nothing to do with Pants, how about deriving the code from pantsbuild instead, which suggests PB.

Reply:
Ah, great suggestion!